__tests__/lib/http/proxy-agent.js

```diff
@@ -90,11 +102,7 @@ describe( 'validate CreateProxyAgent', () => {
 			urlToHit: 'https://wpAPI.org/api',
 		},
 	] )( 'should return null with %o', async ( { envVars, urlToHit } ) => {
-		setEnvironmentVariabeles( envVars );
-		// We have to dynamically import the module so we can set environment variables above
-		// All tests must be async to support this dynamic import, otherwise the modified env variables are not picked up
-		const createProxyAgent = ( await import( '../../../src/lib/http/proxy-agent' ) )
-			.createProxyAgent;
```

Comment:
It magically works :-) I found no code in `createProxyAgent` to cache the old environment variables. And, as far as I understand, multiple `await import` (= `require()`) will load the module only once anyway.